src/Culling/babylon.boundingSphere.ts

```diff
@@ -26,6 +27,21 @@
         public reConstruct(min: Vector3, max: Vector3) {
             this.minimum = min.clone();
             this.maximum = max.clone()
+            
+            var _min = Number.POSITIVE_INFINITY;
+            var _max = Number.NEGATIVE_INFINITY;
+            var _va = ['x','y','z'];
+    
+            for(var i=0; i<3; i++){
+                if(min[_va[i]]<_min){

```

Comment:
this seems wrong to me: the distance cannot be on different axis. How the distance between x and z could work for isntance? we have to evaluate the overall distance